DOC-1415: Identify existing REST API Examples in RS docs and simplify the presentation (#1938)
RedisLabs/redislabs-docs: content/rs/administering/troubleshooting/disabling-services.md

In most deployments, either all of these services are required,
or there are enough memory resources on the nodes for the database requirements.

> In a deployment with limited memory resources, certain services can be turned off from API endpoint to free system memory or using the `rladmin` command.
> Before you turn off a service, make sure that your deployment does not depend on that service.
> After you turn off a service, you can re-enable in the same way.

Reviewer: ```suggestion
In a deployment with limited memory resources, you can use the REST API or `rladmin` to turn off certain services to free system memory.
Before you turn off a service, make sure that your deployment does not depend on that service.
After you turn off a service, you can re-enable it in the same way.
```
